web/src/pages/Environments/EnvironmentForm.tsx

```diff
@@ -0,0 +1,75 @@
+import {Button, Form, FormInstance, Input} from 'antd';
+import {useEffect} from 'react';
+import styled from 'styled-components';
+import RequestDetailsHeadersInput from '../../components/CreateTestPlugins/Rest/steps/RequestDetails/RequestDetailsHeadersInput';
+import {
+  IEnvironment,
+  useCreateEnvironmentMutation,
+  useLazyGetEnvironmentSecretListQuery,
+} from '../../redux/apis/TraceTest.api';
+import {EnvironmentState} from './EnvironmentState';
+
+export const Footer = styled.div`
+  display: flex;
+  justify-content: flex-end;
+  gap: 8px;
+`;
+
+interface IProps {
+  form: FormInstance<IEnvironment>;
+  state: EnvironmentState;
+  onCancel: () => void;
+}
+
+export const EnvironmentForm: React.FC<IProps> = ({state, form, onCancel}) => {
```

Comment:
Can we move this form away from the page folder into a component one? we are starting to think about how to restructure that folder but for the moment I think it is better to have this form elsewhere.

Reply:
ok ✅